src/i_sound.c

```diff
@@ -607,13 +607,20 @@ struct {
 // I_InitSound
 //
 
+int snd_hrtf;
+
 void I_InitSound(void)
 {
     const ALCchar *name;
     ALCdevice *device;
     ALCcontext *context;
     ALCint srate = -1;
 
+    ALCint attribs[] = {
```

Comment:
Could you add a comment that this is a "zero terminated list of integer pairs"? I choked on it at first.